src/Components/SimpleModal/SimpleModal.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { StyleSheet, TouchableOpacity, View } from "react-native";
+import Modal, { ModalProps } from "react-native-modal";
+import getShadowStyles from "src/Themes/getShadowStyle";
+import { isWeb } from "src/Helpers/platform";
+import Colors from "src/Themes/Colors";
+
+const documentBody = document.getElementsByTagName("body")[0];
+
+interface IProps {
+  isVisible: ModalProps["isVisible"];
+  children: ModalProps["children"];
+  onBackdropPress?: ModalProps["onBackdropPress"];
+  onModalShow?: ModalProps["onModalShow"];
```

Comment:
And it is better to provide some props to customize the styles of the modal.

For example, what should a consumer of your lib do, if he wants padding to be not "5%", but "50px"?